examples/converters.py

```diff
@@ -0,0 +1,94 @@
+# This example requires the 'members' privileged intent to use the Member converter.
+
+import typing
+
+import discord
+from discord.ext import commands
+
+intents = discord.Intents.default()
+intents.members = True
+
+bot = commands.Bot("!", intents=intents)
+
+
+@bot.command()
+async def userinfo(ctx: commands.Context, user: discord.User):
+    # In the command signature above, you can see that the `user`
+    # parameter is typehinted to `discord.User`. This means that
+    # during command invocation we will attempt to convert
+    # the value passed as `user` to a `discord.User` instance.
+
+    # If the conversion is successful we will have a User instance
+    # and can do the following:
+    await ctx.send(user.mention)
+
+@userinfo.error
+async def userinfo_error(ctx: commands.Context, error: Exception):
+    # if the conversion above fails for any reason, it will raise `commands.errors.BadArgument`
+    # so we handle this in this error handler
+    if isinstance(error, commands.BadArgument):
+        return await ctx.send("Failed to convert the argument for `user` to `discord.User`.")
+    else:
+        return await ctx.send("Unhandled error: {}".format(error))
```

Comment:
```suggestion
    if isinstance(error, commands.BadArgument):
        return await ctx.send("Couldn't find that user.")
    else:
        # Raise the error to the global error handler
        raise error
```
I think that the example should be more accurate to how users will use it.